### PR TITLE
Remove Unused Analytics events

### PR DESCRIPTION
**Pull request by milenbteam**
## Problem
*Document current behaviour and why it's undesirable here.*

## Solution
*Document how the behaviour was fixed here.*

### Corresponding Branch
*Add link to crdschurch/repo-name-here#issue-number (if applicable). This helps the code reviewer know that corresponding work exists and where to find it.*

## Testing
*Include information on how to test your work here (if applicable).*

### CODE REVIEW

File: _assets/javascripts/views/live-stream.js [hunk 1]
```diff
@@ -14,91 +14,83 @@ function liveStreamInit() {
...
     // These events are deprecated.
-    el.addEventListener('success', function (event) {
-      successInterval = window.setInterval(trackLocation.bind(null, event), 100);
-    });
-    el.addEventListener('dismissed', function (event) {
-      analytics.track('RollCallDismissed', {
-        Source: 'Crossroads.net'
+
+    el.addEventListener("dismissed", function(event) {
+      analytics.track("RollCallDismissed", {
```
Comment (sscarberry) [inline _assets/javascripts/views/live-stream.js:70]: This event was missed in the original refinement of the Card. Could we also remove the "RollCallDismissed" event?

**Comment by bbbtttiii** [comment]
@milenbteam 

I found a few more references to `ChaserView` that could be removed from these files in the `crds-net` repo:

Screen Shot 2023-10-20 at 12 19 57 PM

**Comment by milenbteam** [comment]
> @milenbteam
> 
> I found a few more references to `ChaserView` that could be removed from these files in the `crds-net` repo:
> 
> Screen Shot 2023-10-20 at 12 19 57 PM
> There are also a handful of `give-now-giving` events. These are all in Contentful pages (indicated by the `_site` and `collections` directories) and will need to be removed there:
> 
> Screen Shot 2023-10-20 at 12 44 49 PM



> @milenbteam
> 
> I found a few more references to `ChaserView` that could be removed from these files in the `crds-net` repo:
> 
> Screen Shot 2023-10-20 at 12 19 57 PM
> There are also a handful of `give-now-giving` events. These are all in Contentful pages (indicated by the `_site` and `collections` directories) and will need to be removed there:
> 
> Screen Shot 2023-10-20 at 12 44 49 PM

I don't found give-now-giving in my data. I think that I removed it from Contentful.

**Comment by milenbteam** [comment]
> @milenbteam
> 
> I found a few more references to `ChaserView` that could be removed from these files in the `crds-net` repo:
> 
> Screen Shot 2023-10-20 at 12 19 57 PM
> There are also a handful of `give-now-giving` events. These are all in Contentful pages (indicated by the `_site` and `collections` directories) and will need to be removed there:
> 
> Screen Shot 2023-10-20 at 12 44 49 PM

I don't found give-now-giving in my data. I think that I removed it from Contentful.

**Review by bbbtttiii — APPROVED** [review]

**Pull request merged**